source/_integrations/iron_os.markdown

```diff
@@ -46,7 +60,93 @@ The **IronOS** {% term integration %} seamlessly connects Home Assistant with PI
 - **Uptime:** Tracks the total operating time of the soldering iron since its last power-on.
 - **Hall effect strength:** (Optional) This feature requires the device to be [modified with a hall effect sensor](https://wiki.pine64.org/wiki/Pinecil_Hall_Effect_Sensor). If a neodymium magnet is attached to the stand, it enables proximity detection and can be utilized to calibrate when the iron enters sleep mode based on its proximity to the stand.
 
-
 ## Update
 
-- **Firmware:** The update entity indicates if the firmware is up-to-date or if there is a newer IronOS version available for your device. For more information on how to update your device, please refer to the [IronOS documentation](https://ralim.github.io/IronOS/).
\ No newline at end of file
+- **Firmware:** The update entity indicates if the firmware is up-to-date or if there is a newer IronOS version available for your device. For more information on how to update your device, please refer to the [IronOS documentation](https://ralim.github.io/IronOS/).
+
+## Automations
+
+Get started with this automation example for IronOS with a ready-to-use blueprint!
+
+### Soldering fume extractor automation
+
+Automatically activate the fume extractor when soldering begins and deactivate it when the soldering iron is idle.
+
+{% my blueprint_import badge blueprint_url="https://community.home-assistant.io/t/ironos-soldering-fume-extractor-automation-pinecil-v2/802156" %}
+
+{% details "Example YAML configuration" %}
+
+{% raw %}
+
+```yaml
```

Comment:
Please remove default values, and make sure it follows our YAML style guidelines: https://developers.home-assistant.io/docs/documenting/yaml-style-guide